config/default.json

```
@@ -21,7 +21,7 @@
     },
     "console": {
       "showErrorOnProduction" : false,
-      "verbosityLevel": "only-errors"
+      "verbosityLevel": "display-everything"
```

```suggestion
      "verbosityLevel": "only-errors"
```
i believe by default there should be only errors by default, for debugging it should be changed in local.json because consle logs are bad for performance

---

I insist of display-everything. It’s just for development mode (I double checked). In production no messages are displayed anyway. It helps with debugging a lot. I can’t live without it :) I belive that many newcomers don’t don’t the way to enable the verbosity required for convenient development so let’s make it default